habitat_baselines/utils/common.py

```diff
@@ -101,18 +101,12 @@ def __init__(
         super().__init__()
 
         self.action_activation = config.action_activation
-        self.use_log_std = config.use_log_std
```

Comment:
If we are removing `use_log_std` from the code, can you remove it from the default config as well ?

Reply:
nevermind, I see you removed it.